src/components/doc/layouts.js

```diff
@@ -0,0 +1,19 @@
+import sidebar from '../../../content/docs/docs-nav.yaml';
+
+export const DOCS_LAYOUTS = [{
+  tabLabel: 'Getting started',
+  startPath: '/docs/getting-started/install-github-app/',
+  isActiveMatch: /\/docs\/getting-started/,
+  sidebarNavItemGroups: [sidebar.nav[0]],
+}, {
+  tabLabel: 'Plugins & Integrations',
+  startPath: '/docs/integrations/argocd/',
+  isActiveMatch: '/docs/(?:custom-plugins|integrations)/',
```

Comment:
Would it make sense to add a redirect to the root of `/docs/integrations` and `/docs/custom-plugins` (and others) to point to argocd instead of hardcoding the first `to` value here? 
Currently the path to those top level URLs returns 404.

Reply:
That would make the code look like this right? I wouldn't be able to get rid of the hardcoded value, but it would be a less specific hardcoded value tan it currently is.

```
tabLabel: 'Plugins & Integrations',
startPath: '/docs/integrations/',
isActiveMatch: '/docs/(?:custom-plugins|integrations)/',
```

I do agree that `/docs/integrations` and `/docs/custom-plugins` should not 404. I've made [this ticket](https://app.shortcut.com/larder/story/7997/make-an-integrations-homepage) to fix that. Until that ticket is delivered, perhaps we can live with `startPath: '/docs/integrations/argocd/',`?